app/i18n/zh-tw/admin.php

```diff
@@ -195,8 +195,8 @@
 		'changelog' => 'Changelog',	// TODO
 		'check' => '檢查更新',
 		'copiedFromURL' => 'update.php copied from %s to ./data',	// TODO
-		'current_version' => '當前 FreshRSS 版本為 %s。',
-		'last' => '上次檢查：%s',
+		'current_version' => '當前 版本為',
```

Comment:
Pinging @den13501 for review

Reply:
The line above is more important than this mentioned line ;)